File: src/sdk-reference/index/code-example/exists.js
kuzzle
  .index
  .exists('nyc-open-data')
  .then(result => {
    if (result === true) {
      console.log('index exists')
    } else {
      console.log('index does not exist')
    }
  })
  .catch(error => {
    console.error(error.message)
  });

File: src/sdk/cpp/1/controllers/index/exists/snippets/exists.cpp
try {
  bool exists = kuzzle->index->exists("nyc-open-data");

  if (exists) {
    std::cout << "Index exists in Kuzzle" << std::endl;
  }
} catch (kuzzleio::KuzzleException &e) {
  std::cerr << e.what() << std::endl;
}
